### DISCUSSION TITLE
Chia Blockchain

### DISCUSSION THREAD

**Discussion by Timskies** [Chia-Network/chia-blockchain#6732] [Support]
does anyone know how to fix the bug that chia is  not connecting  with my wallet

note:  i already  tried reinstall, reloading and restarting

**Comment by loppefaaret** [comment]
It can happen that the wallet database has been corrupted:
WALLET DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
del ~\.chia\mainnet\wallet\db\block*
```
Then try opening up the Chia app again

**Comment by BrandtH22** [comment]
@Timskies , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
